packages/app-store/routing-forms/config.json

```diff
@@ -2,13 +2,15 @@
   "/*": "Don't modify slug - If required, do it using cli edit command",
   "name": "Routing Forms",
   "title": "Routing Forms",
+  "isGlobal": true,
```

Comment:
Make Routing Forms app preinstalled.

Reply:
Would that mean that this URL should work? Or am I misunderstanding this?
![Screenshot 2023-07-26 at 12 14 59](https://github.com/calcom/cal.com/assets/30310907/62081e04-e811-484e-8a9a-68543dc2b29b)